src/views/TimezoneModal.vue

```diff
@@ -16,7 +16,8 @@
   <ion-content class="ion-padding">
     <!-- Empty state -->
     <div class="empty-state" v-if="filteredTimeZones.length === 0">
-      <p>{{ $t("No time zone found")}}</p>
+      <ion-spinner name="crescent" ></ion-spinner>
+      <p>{{ $t("Fetching time zones") }}</p>
```

Comment:
The spinner and text should be displayed in the same line as present in the figma.